src/allmydata/frontends/magic_folder.py

```diff
@@ -955,7 +952,7 @@ def _write_downloaded_file(self, local_path_u, abspath_u, file_contents,
             return self._rename_conflicted_file(abspath_u, replacement_path_u)
         else:
             try:
-                fileutil.replace_file(abspath_u, replacement_path_u, backup_path_u)
+                fileutil.replace_file(abspath_u, replacement_path_u)
```

Comment:
Okay that's really straightforward.  Replace without backup instead of replace with backup.